geomtwo/Vector bearing_measurement
bool truemeasure
float32 last_beta